source/dotnet/Library/AdaptiveCards.Rendering.Html/AdaptiveCardRenderer.cs

```diff
@@ -685,7 +688,7 @@ protected static HtmlTag ChoiceSetRender(AdaptiveChoiceSetInput adaptiveChoiceSe
                         .Attr("name", adaptiveChoiceSetInput.Id)
                         .AddClass("ac-input")
                         .AddClass("ac-multichoiceInput")
-                        .Style("width", "100%");
+                        .Style("width", "100%"); 
```

Comment:
Did you accidently add a trailing space here? That's what GitHub seems to indicate